app/src/main/res/values/strings.xml

```diff
@@ -1398,7 +1394,7 @@
     <string name="profile_config_name_hint">Nazwa profilu</string>
     <string name="profile_config_sync_enabled">Synchronizuj ten profil</string>
     <string name="profile_config_logout">Wyloguj się</string>
-    <string name="permissions_qr_scanner">Aby móc zeskanować kod QR musisz przyznać uprawnienia dostępu do kamery.\n\nKliknij OK, aby przyznać uprawnienia.</string>
+    <string name="permissions_qr_scanner">Aby móc zeskanować kod QR musisz przyznać uprawnienia dostępu do kamery. \\n\\nKliknij OK, aby przyznać uprawnienia.</string>
```

Comment:
If you escape the `\n` with double backslash, it becomes visible as `\n` in the UI, we don't want that. Just undo any changes to this line.